packages/components/pages/f-checkout/src/tenants/it-IT.js

```diff
@@ -1,4 +1,318 @@
+                text: 'Lascia una nota per il ristorante se ci sono informazioni particolari di cui devono essere a conoscenza. Non inserire dettagli sulle allergie.',
+                placeholder: 'ad es. il citofono non funziona. Non inserire dettagli sulle allergie in questo campo. Insieme ai ristoranti, ci stiamo impegnando per ridurre i rifiuti: aiutaci anche tu evitando di chiedere posate in plastica.'
+            },
+            dinein: {
+                title: 'Lascia una nota',
+                text: 'Lascia una nota per il ristorante se ci sono informazioni particolari di cui devono essere a conoscenza. Non inserire dettagli sulle allergie.',
+                placeholder: 'Non inserire dettagli sulle allergie in questo campo. Insieme ai ristoranti, ci stiamo impegnando per ridurre i rifiuti: aiutaci anche tu evitando di chiedere posate in plastica.'
+            }
+        },
+        kitchen: {
+            delivery: {
+                title: 'Aggiungi una nota per la cucina',
+                text: 'Lascia una nota per il ristorante se ci sono informazioni particolari di cui devono essere a conoscenza.',
+                placeholder: 'Insieme ai ristoranti, ci stiamo impegnando per ridurre i rifiuti: aiutaci anche tu evitando di chiedere posate in plastica.'
+            },
+            dinein: {
+                title: 'Aggiungi una nota per la cucina',
+                text: 'Lascia una nota per il ristorante se ci sono informazioni particolari di cui devono essere a conoscenza.',
+                placeholder: 'Insieme ai ristoranti, ci stiamo impegnando per ridurre i rifiuti: aiutaci anche tu evitando di chiedere posate in plastica.'
+            },
+            collection: {
+                title: 'Aggiungi una nota per la cucina',
+                text: 'Lascia una nota per il ristorante se ci sono informazioni particolari di cui devono essere a conoscenza.',
+                placeholder: 'Insieme ai ristoranti, ci stiamo impegnando per ridurre i rifiuti: aiutaci anche tu evitando di chiedere posate in plastica.'
+            }
+        }
+    },
+
+    checkoutHeader: {
+        user: {
+            title: '{name}, conferma i tuoi dati',
+            switchUser: 'Non sei {name}? Clicca qui'
+        },
+        guest: {
+            loginTitle: 'Come desideri procedere?',
+            loginButton: 'Come desideri procedere?',
+            option: 'o',
+            guestTitle: 'Continua senza creare un account',
+            confirmation: 'Per favore, conferma i tuoi dati {serviceType}'
+        }
+    },
+
+    serviceTypes: {
+        collection: 'Ritiro',
+        delivery: 'Consegna'
+    },
+
+    checkoutTermsAndConditions: 'Effettuando un ordine, accetti i nostri {termsAndConditions}. Leggi la nostra {privacyPolicy} e la {cookiePolicy}.',
+
+    termsAndConditionsLinkText: 'Termini & Condizioni',
+    termsAndConditionsLinkUrl: 'https://www.just-eat.es/info/terminos-y-condiciones',
```

Comment:
Is this correct? points to `es` instead of `it`?

Reply:
yes, yes it should 🤦 Thank you!